rails/Dockerfile

```diff
@@ -52,6 +53,8 @@ WORKDIR $APP_HOME
 
 USER app
 
+RUN gem install json -v 2.6.3
```

Comment:
Why do we need to install this outside of the `Gemfile`?

Reply:
Thanks for catching that. That should not have been left there. I've removed it.